module/zfs/dsl_crypt.c

```diff
@@ -2105,6 +2097,8 @@ dsl_crypto_recv_raw_objset_sync(dsl_dataset_t *ds, dmu_objset_type_t ostype,
 	 */
 	arc_release(os->os_phys_buf, &os->os_phys_buf);
 	bcopy(portable_mac, os->os_phys->os_portable_mac, ZIO_OBJSET_MAC_LEN);
+	os->os_phys->os_flags &= ~OBJSET_FLAG_USERACCOUNTING_COMPLETE;
+	os->os_flags = os->os_phys->os_flags;
```

Comment:
do we not need to unset OBJSET_FLAG_USEROBJACCOUNTING_COMPLETE as well?

Reply:
Makes sense, will fix.